heartbeat.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "centre": {
+      "description": "Name of centre",
+      "type": "string",
+      "pattern": "^[a-z]{3,30}$"
+    },
+    "male": {
```

Comment:
Should we be explicit in the key that this count is "occupied male" (or female) for the avoidance of any doubt? How do we deal with beds that are n/a gender?